perfkitbenchmarker/benchmarks/silo_benchmark.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
The template boilerplate header (hooks/boilerplate.py.txt) includes that line, and so the pre-commit hook (hooks/boilerplate.sh) complains if I don't include it.